src/netgroup.h

```diff
@@ -69,8 +82,17 @@ class NetGroupManager {
      * re-bucketed.
      *
      * This is initialized in the constructor, const, and therefore is
-     * thread-safe. */
-    const std::vector<bool> m_asmap;
+     * thread-safe. m_asmap can either point to m_loaded_asmap which holds
+     * data loaded from an external file at runtime or it can point to embedded
+     * asmap data.
+     */
+    const std::span<const std::byte>  m_asmap;
+    std::vector<std::byte> m_loaded_asmap;
```

Comment:
TODO: Figure out why
```suggestion
    const std::vector<std::byte> m_loaded_asmap;
```
Leads to addrman failures in feature_asmap.py

Reply:
I didn't fully validate this yet but it looks to me like with the const the vector can not be moved anymore, it is copied instead. This means that when the ctor `WithLoadedAsmap` tries to move it, it is copied instead and then the span pointer is dangling.